primitives/src/jobs/tss.rs

```diff
@@ -199,4 +199,7 @@ pub enum DigitalSignatureScheme {
 
 	/// BLS 381 signature scheme.
 	Bls381,
+
+	/// WSTS V2 signature scheme.
+	WstsV2,
```

Comment:
I overlooked this. This should not be used. WSTS uses `secp256k1` according to their readme and so we should just use `SchnorrSecp256k1`.